cmd/describe/cluster/cmd.go

```diff
@@ -70,6 +70,12 @@ func run(cmd *cobra.Command, argv []string) {
 	cluster := r.FetchCluster()
 	isHypershift := cluster.Hypershift().Enabled()
 
+	displayName := ""
+	subscription, subscriptionExists, _ := r.OCMClient.GetSubscriptionByID(cluster.ID())
```

Comment:
Would it be worth handling the error, even just in debug mode? Not worth exiting if there is an error on this, but at least we can be sure to print some logs in debug mode.

Reply:
--debug will log the request & response anyway, no?

Reply:
added debug output, thanks!